src/Storefront/Theme/Autoprefixer.php

```diff
@@ -1,41 +0,0 @@
-<?php declare(strict_types=1);
```

Comment:
Can we delete this class or do we have to deprecate it for `6.4` and keep it for now? @shyim

Reply:
As we failed to mark it as `@internal`, I would deprecate it :see_no_evil:

Reply:
I added a deprecation notice and restored the class (which is not used anymore).